src/ks-app/src/app/app.module.ts

```diff
@@ -78,8 +79,10 @@ import {KSWizards} from "./containers/wizard/wizards.component";
         KSKillersDemo,  KSHotFussDemo,    KSProgressBars,  KSSpinners,
         KSTables,       KSTooltips
     ],
-    imports:
-        [BrowserModule, BrowserAnimationsModule, FormsModule, ReactiveFormsModule, ClarityModule, AppRoutingModule],
+    imports: [
+        BrowserModule, BrowserAnimationsModule, FormsModule, ReactiveFormsModule, ClarityModule, ClrRadioModule,
```

Comment:
You should also import Checkboxes and add them to KS app

Reply:
I didn't because I was thinking of a checkbox conflict, but I guess I can do it now.